### ISSUE TITLE
How to implement FFT in loss function

### ISSUE THREAD

**Issue by kingaza** [fchollet/keras#5649]
I need to do 2D FFT for the images in loss function for training. However, I don't see any example for it. 
Could you give me a hint? thx

**Comment by unrealwill** [comment]
Hello,

You probably want to use tensorflow and use tf.fft2d
In your custom loss fonction.

**Comment by Dref360** [comment]
Warning : tf.fft2d is not supported for CPU

**Comment by kingaza** [comment]
thx @unrealwill @Dref360 
I will try later.
